AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -275,23 +274,22 @@ - (void)downloadConfigurationWithETag:(nullable NSString *)eTag {
                 if (configUrl) {
                   NSString *newETag = [MSHttpIngestion eTagFromResponse:response];
                   if (newETag) {
-                    [MS_USER_DEFAULTS setObject:newETag forKey:kMSIdentityETagKey];
+                    [MS_USER_DEFAULTS setObject:newETag forKey:kMSAuthETagKey];
                   }
                 } else {
-                  MSLogWarning([MSIdentity logTag], @"Couldn't create Identity config file.");
+                  MSLogWarning([MSAuth logTag], @"Couldn't create Auth config file.");
                 }
                 @synchronized(self) {
-                  self.identityConfig = config;
+                  self.authConfig = config;
 
                   // Reinitialize client application.
                   [self configAuthenticationClient];
                 }
               } else {
-                MSLogError([MSIdentity logTag], @"Downloaded identity configuration is not valid.");
+                MSLogError([MSAuth logTag], @"Downloaded auth config is not valid.");
```

Comment:
Not on you but since I saw it we could harmonize this one with the others:
```suggestion
                MSLogError([MSAuth logTag], @"Downloaded Auth config is not valid.");
```

Reply:
I think this shouldn't be capitalized since it's not talking about that auth service

Reply:
Yep. I think so too.